webgui-new/src/server.ts

```diff
@@ -16,9 +16,10 @@ const main = async () => {
   const proxyHandler = createProxyMiddleware({
     target: process.env.BACKEND_URL,
     changeOrigin: true,
-    pathFilter: '**/*Service',
+    pathFilter: ['**/*Service', '**/*ga.txt'],
```

Comment:
Maybe it is enough to filter for `ga.txt` files, and not all files ending with that suffix.

Reply:
Fair point. Fixed.